force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,7 +35,17 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+
+        Integer result;
+
+        if (a == null || b == null) {
+            result = null;
+        }
+        else {
```

Comment:
This compiles, but it's preferred to maintain spacing and line break standards in Apex, as follows:

} else {

Reply:
Will do. Have a C/C# background so some brace habits hard to break lol